ui.frontend/src/constants.js

```diff
@@ -38,34 +38,34 @@ export const Constants = {
     FORM_CONTAINER_INITIALISED : "AF_FormContainerInitialised",
 
     /**
-     * @summary Panel event to be triggered when a child view is added to panel view
+     * @summary Panel event to be triggered when a panel instance view is added
      *
-     * @name  AF_PanelChildAdded
+     * @name  AF_PanelInstanceAdded
      * @event
      * @property {object} event
      * @property {object} event.detail instance of child view that is added
      * @example
-     * panel.element.on("AF_PanelChildAdded" , function(event) {
+     * formcontainer.getFormElement().on("AF_PanelInstanceAdded" , function(event) {
```

Comment:
this should be exposed via GuideBridge also as guidebridge is the only mechanism for external custom code (outside of component runtime)

Reply:
we will do it as part of GuideBridge API parity. not required as part this PR though

Reply:
This is not something which external code will consume. This is something which will be consumed with in form via rules.